emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -122,9 +154,12 @@
 
     variantPopup = new EmojiVariantPopup(this.rootView, clickListener);
 
-    final EmojiView emojiView = new EmojiView(context, clickListener, longClickListener, recentEmoji, variantEmoji, backgroundColor, iconColor, dividerColor);
+    final EmojiView emojiView =
+        new EmojiView(context, clickListener, longClickListener, recentEmoji, variantEmoji,
+            backgroundColor, iconColor, dividerColor);
```

Comment:
Or this. When first looking at this, it suggests an actual change has been made to the `EmojiView` constructor or it's parameters.

Reply:
All good, I understand the pain.